packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -839,39 +872,44 @@ test('authorized group is allowed to listen to onUpdate', async () => {
         }
       }
     `,
-    fetchPolicy: 'network-only',
-  });
+    authMode: GRAPHQL_AUTH_MODE.AMAZON_COGNITO_USER_POOLS,
+  }) as Observable<any>;
 
   const subscriptionPromise = new Promise((resolve, reject) => {
     const subscription = observer.subscribe({
       next: event => {
-        subscription.unsubscribe();
-        const subResponse = event.data.onUpdateMember;
+        const subResponse = event.value.data.onUpdateMember;
         subscription.unsubscribe();
         expect(subResponse).toBeDefined();
         expect(subResponse.id).toEqual(memberID);
-        expect(subResponse.name).toEqual(memberName);
+        expect(subResponse.name).toEqual(newMemberName);
         resolve(undefined);
       },
       complete: () => {},
       error: err => {
-        console.log(err);
-        reject(err);
+        console.log(JSON.stringify(err, null, 4));
```

Comment:
Is this a normal JavaScript error? If so, it won't stringify like a normal object.

Reply:
The error is not a a JS object but looks something like this
```
  thrown: Object {
      "error": Object {
        "errors": Array [
          [Object],
        ],
      },
      "provider": AWSAppSyncRealTimeProvider {
        "_config": Object {
          "Auth": [Object],
        },
        "awsRealTimeSocket": null,
        "keepAliveTimeout": 300000,
        "keepAliveTimeoutId": Timeout {
          "_destroyed": true,
          "_idleNext": null,
          "_idlePrev": null,
          "_idleStart": 238082,
          "_idleTimeout": -1,
          "_onTimeout": null,
          "_repeat": null,
          "_timerArgs": undefined,
          Symbol(refed): true,
          Symbol(asyncId): 4880,
          Symbol(triggerId): 4862,
        },
        "promiseArray": Array [],
        "socketStatus": 0,
        "subscriptionObserverMap": Map {},
      },
    }
    ```